andela/jubilee-backend
# #167733640 Setup backend application to use docker

#### Description of Task to be completed?
Should ensure that docker is properly setup on this application
#### How should this be manually tested?
On the project root directory, run `docker-compose up -d` this should start a local development server with all the necessary services running on the background
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#167733640
#### Screenshots (if appropriate)
![Screen Shot 2019-08-15 at 1 34 24 AM](https://user-images.githubusercontent.com/33099347/63065158-ea182e00-befc-11e9-917c-d41821cdb925.png)
